test/h5test.c

```diff
@@ -114,11 +114,15 @@ const char *LIBVER_NAMES[] = {"earliest", /* H5F_LIBVER_EARLIEST = 0  */
 static H5E_auto2_t err_func = NULL;
 
 /* Global variables for testing */
-size_t   n_tests_run_g     = 0;
-size_t   n_tests_passed_g  = 0;
-size_t   n_tests_failed_g  = 0;
-size_t   n_tests_skipped_g = 0;
-uint64_t vol_cap_flags_g   = H5VL_CAP_FLAG_NONE;
+static int TestExpress_g     = -1; /* Whether to expedite testing. -1 means not set yet. */
+size_t     n_tests_run_g     = 0;
+size_t     n_tests_passed_g  = 0;
+size_t     n_tests_failed_g  = 0;
+size_t     n_tests_skipped_g = 0;
+uint64_t   vol_cap_flags_g   = H5VL_CAP_FLAG_NONE;
+
+/* Whether h5_cleanup should clean up temporary testing files */
+static bool do_test_file_cleanup_g = true;
```

Comment:
While this duplicates the `GetTestCleanup()` functionality from testframe.c into h5test.c, it prevents conflating the two functionalities. h5test.c's usage was also less structured (the application had to call `h5_fixname` to influence the status), so it seemed best to leave them separate for now until a potential unified testing framework can come together in the future.